package.json

```diff
@@ -18,33 +18,37 @@
     "test": "ember test"
   },
   "devDependencies": {
-    "@ember/jquery": "^0.5.2",
-    "@ember/optional-features": "^0.6.3",
-    "broccoli-asset-rev": "^2.7.0",
-    "ember-ajax": "^4.0.1",
-    "ember-cli": "~3.8.1",
+    "babel-eslint": "^8.0.2",
+    "@ember/optional-features": "^0.7.0",
+    "@glimmer/component": "^0.14.0-alpha.3",
+    "broccoli-asset-rev": "^3.0.0",
+    "ember-auto-import": "^1.2.20",
+    "ember-ajax": "^5.0.0",
+    "ember-cli": "github:ember-cli/ember-cli#b0e01db47ae7e369d4acb937d0d580dae80e9d84",
     "ember-cli-app-version": "^3.2.0",
-    "ember-cli-babel": "^7.1.2",
-    "ember-cli-dependency-checker": "^3.1.0",
-    "ember-cli-eslint": "^4.2.3",
+    "ember-cli-babel": "^7.7.0",
+    "ember-cli-dependency-checker": "^3.0.0",
+    "ember-cli-eslint": "^5.0.0",
     "ember-cli-htmlbars": "^3.0.0",
-    "ember-cli-htmlbars-inline-precompile": "^1.0.3",
-    "ember-cli-inject-live-reload": "^1.8.2",
-    "ember-cli-mirage": "^0.4.9",
+    "ember-cli-htmlbars-inline-precompile": "^2.0.0",
+    "ember-cli-inject-live-reload": "^2.0.1",
+    "ember-cli-mirage": "^0.4.15",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-template-lint": "^1.0.0-beta.1",
     "ember-cli-tutorial-style": "^2.0.0",
     "ember-cli-uglify": "^2.1.0",
     "ember-cli-update": "^0.32.4",
-    "ember-data": "~3.8.0",
+    "ember-data": "^3.10.0",
     "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^1.1.0",
+    "ember-load-initializers": "^2.0.0",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-qunit": "^3.4.1",
-    "ember-resolver": "^5.0.1",
+    "ember-qunit": "^4.1.2",
+    "ember-resolver": "^5.1.3",
     "ember-simple-leaflet-maps": "^0.0.2",
```

Comment:
I think if we update this, we shouldn't need jQuery (unless there is another plugin that is requiring it)

Crosslinking from https://github.com/ember-learn/ember-simple-leaflet-maps/issues/2